paper/README.Rmd

```diff
@@ -90,7 +90,7 @@ There has been much research on mode shift since the origins of applied transpor
 Likewise, getting people walking is worthwhile on environmental [@cervero_travel_1997; @frank_impacts_1994], community cohesion and health grounds [@handy_critical_2005; @handy_how_2002].
 Recently there has been an increase in research activity on various pedestrian models supporting uptake of walking [@aoun_bicycle_2015; @cooper_using_2019; @ewing_varying_2014; @griswold_pedestrian_2019; @kuzmyak_estimating_2014; @martinez-gil_modeling_2017; @munira_use_2017; @turner_synthesis_2017].
 
-Recent policy interest has been shown in planning active transportation networks for specific destinations, such as schools and major employers.
+Recent policy interest has been shown in planning active transportation networks for specific destinations, such as schools and major employers [@larouche_built_2015; @uttley_cycling_2016].
```

Comment:
Heads-up @fiftysevendegreesofrad this addresses the comment in the docx "Robin do we have a citation to back this up?"

Reply:
Cool. I'm slightly confused why I'm getting the merge request as it's your repo?